### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by felixrieseberg**
This PR updates various dependencies, most notably Electron from v7 to v8.

### CODE REVIEW

File: static/releases.json [hunk 1]
```diff
@@ -1 +1 @@
...
...rsion":"7.0.0-beta.4"},{"version":"6.0.8"},{"version":"6.0.9"},{"version":"6.0.10"},{"version":"4.2.11"},{"version":"7.0.0-beta.5"},{"version":"5.0.11"},{"version":"6.0.11"},{"version":"6.0.12"},{"version":"7.0.0-beta.6"},{"version":"7.0.0-beta.7"},{"version":"4.2.12"},{"version":"7.0.0"},{"version":"6.1.0"},{"version":"6.1.1"},{"version":"8.0.0-beta.1"},{"version":"6.1.2"},{"version":"8.0.0-beta.2"},{"version":"6.1.3"},{"version":"7.0.1"},{"version":"5.0.12"},{"version":"6.1.4"},{"version":"7.1.0"},{"version":"7.1.1"},{"version":"7.1.2"},{"version":"8.0.0-beta.3"},{"version":"6.1.5"},{"version":"7.1.3"},{"version":"8.0.0-beta.4"},{"version":"7.1.4"},{"version":"6.1.6"},{"version":"7.1.5"},{"version":"6.1.7"},{"version":"5.0.13"},{"version":"7.1.6"},{"version":"8.0.0-beta.5"},{"version":"7.1.7"},{"version":"7.1.8"},{"version":"7.1.9"},{"version":"8.0.0-beta.6"},{"version":"8.0.0-beta.7"},{"version":"7.1.10"},{"version":"8.0.0-beta.8"},{"version":"8.0.0-beta.9"},{"version":"7.1.11"},{"version":"8.0.0"},{"version":"9.0.0-beta.1"},{"version":"7.1.12"},{"version":"8.0.1"},{"version":"9.0.0-beta.2"},{"version":"7.1.13"},{"version":"6.1.8"},{"version":"9.0.0-beta.3"},{"version":"8.0.2"}]
```
Comment (malept) [inline static/releases.json]: Remind me why we don't just add a dependency on [`electron-releases`](https://github.com/electron/releases) ?

Comment (felixrieseberg) [reply static/releases.json]: We don't shake our dependencies and the full `electron-releases` weighs a whopping 48 mb!

Comment (malept) [reply static/releases.json]: That would do it.

Comment (malept) [reply static/releases.json]: I wrote a GitHub Action to automate this for us: https://github.com/electron/fiddle/pull/331

**Pull request merged**